apps/angular/projection/src/app/component/city-card/city-card.component.ts

```diff
@@ -8,24 +10,27 @@ import { CardComponent } from '../../ui/card/card.component';
 @Component({
   selector: 'app-city-card',
   template: `
-    <app-card [list]="cities" [type]="cardType" customClass="bg-light-green">
+    <app-card [list]="cities()" [type]="cardType" customClass="bg-light-green">
       <img src="assets/img/city.png" width="200px" />
     </app-card>
   `,
   standalone: true,
   imports: [CardComponent],
 })
 export class CityCardComponent implements OnInit {
-  cities: City[] = [];
+  cities: WritableSignal<City[]> = <WritableSignal<City[]>>toSignal(
+    this.http.fetchCities$.pipe(
+      tap((c) => this.store.addAll(c)),
+      switchMap(() => this.store.cities$),
+    ),
+  );
```

Comment:
no, i'm not in favor of this type of code. 
leave initialization inside the constructor or ngoninit and get the data from the store with a signal.

you can store the city inside your store as a signal.
but the call to the backend should be outside of this.